File: Algorithms/BOJ/10808.py
# https://www.acmicpc.net/problem/10808
# 알파벳 개수
import sys

word = sys.stdin.readline().strip()
cnt = [0 for _ in range(0, 26)]

for w in word:
    cnt[ord(w) - 97] += 1

print(" ".join(map(str, cnt)))

File: Algorithms/BOJ/Swift/10808.swift
// https://www.acmicpc.net/problem/10808
// 알파뱃 개수

let word = readLine()!.lowercased()
var count = Array(repeating: 0, count: 26)

for char in word.utf16 {
    count[Int(char) - 97] += 1
}

print(count.map { String($0) }.joined(separator: " "))
